src/test/java/com/wojtek/parkingmeter/controllers/TicketControllerCheckChargeTest.java

```diff
@@ -0,0 +1,49 @@
+package com.wojtek.parkingmeter.controllers;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+public class TicketControllerCheckChargeTest {
```

Comment:
Te wszystkie testy (wszystkie klasy) to takie ani testy unitowe (bo dostają się wprost do bazy zamias mockować), ani integracyjne (bo nie testują całego przypadku biznesowego).

Reply:
Nie wiem jak zrobić to inaczej/lepiej.